usaspending_api/disaster/management/commands/generate_covid19_download.py

```diff
@@ -75,7 +75,7 @@ def process_data_copy_jobs(self):
             intermediate_data_file_path = final_name.parent / (final_name.name + "_temp")
             data_file, count = self.download_to_csv(sql_file, final_name, str(intermediate_data_file_path))
             if count <= 0:
-                raise Exception(f"Missing Data for {final_name}!!!!")
+                logger.warning(f"Empty data file generated: {final_name}!")
```

Comment:
I'm happy to see this.  I wanted to ask you about it but didn't want to disturb your PTO.  👍